credentials/apps/credentials_theme_philu/templates/philu/credentials/programs/professional-certificate/certificate.html

```diff
@@ -0,0 +1,59 @@
+{% load staticfiles %}
```

Comment:
Let's please check with product team about `professional-certificate`, just a proactive approach. Just post a message on slack, tell them about all three or four options and what makes more sense in PhilanthropyU's use case.

Reply:
Good catch 👍

Reply:
Configured XSeries for certificate as per discussion with Product Team. Also keep `professional-certificate` as it is being used in unit tests.